Pipfile

```diff
@@ -6,14 +6,15 @@ verify_ssl = true
 [dev-packages]
 black = ">=19"
 isort = ">=4.3"
+pytest = ">=4"
```

Comment:
Did you miss `pipfile`.
I think using `pipenv lock -r > requirements.txt` is a much better approach than using 4 years old package.

Reply:
From experience, `pipenv lock -r > requirements.txt` is not a good idea, since it introduces multiple source of truth and redundancy, which can lead to inconsistency and chaos 🔥

The rest of your comments are fixed in https://github.com/Cognexa/plcx/pull/42